src/interfaces/swig/SGBase.i

```diff
@@ -447,6 +472,25 @@ namespace shogun
 }
 
 %pythoncode %{
+def _internal_get_param(self, name):
+
+	for f in (self._get,
+			self._get_real,
+			self._get_int,
+			self._get_real_matrix,
+			self._get_real_vector,
+			self._get_int_vector
+			):
+		try:
+			return f(name)
+		except (SystemError, RuntimeError):
+			pass
+		except Exception:
+			raise
+	raise KeyError("There is no parameter called '{}' in {}".format(name, self.get_name()))
```

Comment:
I would prefer if we would somehow connect the error msg with the ones present in the c++ lib, where this error already exists. Further, if done as here, the user should be informed that this error is within the interface rather than core shogun.